AdobeStockImageAdminUi/Test/Mftf/Test/UserSeesStockImageAttributesLocalizedTest.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="UserSeesStockImageAttributesLocalizedTest">
+        <annotations>
+            <stories value="User sees stock image attributes localized"/>
+            <title value="Admin should be able to sees stock image attributes localized"/>
+            <description value="Admin should be able to sees stock image attributes localized"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
+           <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="setCorrectModuleConfig"/>
+            <actionGroup ref="LoginActionGroup" stepKey="loginAsAdmin"/>
+            <!--Set Admin "Interface Locale" to ru_RU-->
+            <actionGroup ref="SetAdminAccountActionGroup" stepKey="setAdminInterfaceLocaleToDefaultValue">
+                <argument name="InterfaceLocaleByValue" value="ru_RU"/>
+            </actionGroup>
+        </before>
+        <after>
+            <actionGroup ref="AdminAdobeStockSetConfigActionGroup" stepKey="cleanModuleConfig">
+                <argument name="enabled" value="0"/>
+                <argument name="wysiwygEnabled" value="disabled"/>
+<!--                TODO: uncomment next lines where issue with sensitive config setting will be resolved. -->
+<!--                <argument name="apiKey" value=""/>-->
+<!--                <argument name="privateKey" value=""/>-->
+            </actionGroup>
+            <actionGroup ref="SetAdminAccountActionGroup" stepKey="setAdminInterfaceLocaleToDefaultValue">
+                <argument name="InterfaceLocaleByValue" value="en_US"/>
+            </actionGroup>
+            <amOnPage url="admin/admin/auth/logout/" stepKey="amOnLogoutPage"/>
```

Comment:
There is a specific action group for logout action
`<actionGroup ref="logout" stepKey="logout"/>`